### ISSUE TITLE
Tagmanger should be on the control panel

### ISSUE THREAD

**Issue by danielo515** [Jermolene/TiddlyWiki5#2577]
Hello,

As an experienced user ( using TW for more than 3 years no) I feel very dumb when it takes to me so much time to do things that I consider basic. For example, opening the tag manager always takes me more than a minute. Why? Because my instinct is to look for it on some tab of the control panel. But on the control panel, which is supposed to control your whole wiki, does not even have a reference to the tag manager.

As a control tool, the tag manager should be on the control panel.

Regards

**Comment by gernert** [comment]
@danielo515

You can always copy the following (from $:/core/ui/MoreSideBar/Tags) into a tiddler:

```
<$set name="tv-config-toolbar-icons" value="yes">

<$set name="tv-config-toolbar-text" value="yes">

<$set name="tv-config-toolbar-class" value="">

{{$:/core/ui/Buttons/tag-manager}}

</$set>

</$set>

</$set>
```
and tag it accordingly, e.g. $:/tags/ControlPanel/Appearance

**Comment by tobibeer** [comment]
I tend to agree that I prefer configuration and use to be separate from one another, he tag manager clearly being configuration.

In fact, I would love to see a "simple" toggle that allows to switch between a defined read-only and then an author-mode, whereas that toggle could be hidden so much so that a general visitor might not even know the thing is editable.

**Comment by duarteframos** [comment]
I wholeheartedly agree, the tag manager in its current place feels out of place. I don't use it very often, but when I do I always have to think were to find it.

It also feels non-standard in the sense that it's the only toolbar button that opens an addition tiddler in the river, as opposed to most others that open either a popup or a dropdown menu and mostly contain 'quick tools' or 'contextual actions', and not full administration tools.

**Comment by Jermolene** [comment]
The control panel is for configuring/customising the wiki, and not for operations on the content within the wiki.

Personally, I favour refactoring `$:/TagManager` to be one tab of a broader `$:/Manager` tiddler which would bring together all the ways of manipulating content tiddlers. Other features might include bulk operations, search and replace and export configurations.

@duarteframos - toolbar buttons like advanced search, new tiddler, new journal, new image all open a new tiddler in the story river

**Comment by duarteframos** [comment]
Yes, you are right. A manager tiddler sounds like a more reasonable option indeed.

Batch operations like those and maybe even advanced searches would certainly be at home there.

+1 for this :)

**Comment by danielo515** [comment]
Dear @Jermolene ,

If you want to implement it that way, it would be an improvement. As I said, where it is currently is very hard to find. But I don't understand your point. ControlPanel is some kind of generic word, and I don't see why managing the color of tags (which is just metadata) should be on a different panel.

Regards

**Comment by Jermolene** [comment]
Perhaps the way to think about it is that there is nothing you can do within control panel to affect the values of any of your content tiddlers; literally everything in there is about changing system tiddlers that affect the operation or appearance of TiddlyWiki itself. Meanwhile, the tiddler manager is about changing your own content tiddlers.
